backend/src/main/kotlin/org/loculus/backend/controller/SubmissionController.kt

```diff
@@ -252,23 +242,23 @@ class SubmissionController(
     fun getReleasedData(
         @PathVariable @Valid organism: Organism,
         @RequestParam compression: CompressionFormat?,
+        @RequestHeader(value = HttpHeaders.IF_NONE_MATCH, required = false) ifNoneMatch: String?,
```

Comment:
Can we change this name? `if_changed_since` for example makes it clear that the tag is a timestamp.

Reply:
it doesn't matter whether it's a timestamp or not - if it's not equal, it needs to be refreshed. this keeps it close to etag spec/as vanilla as possible

Reply:
Ah I see this is standard when using e-tags: https://www.geeksforgeeks.org/http-headers-if-none-match/